app/controllers/report_controller.rb

```diff
@@ -18,6 +18,12 @@ def empty_theses
     @record = report.empty_theses_record subset
   end
 
+  def expired_holds
+    report = Report.new
+    holds = Hold.all
```

Comment:
Did you consider using the `active_or_expired` scope here directly?

`holds = Hold.active_or_expired`

That would allow for removing that scope from the report itself and I believe end up with the same result.

Reply:
Yeah, I had this in an earlier iteration and moved away from it. That is definitely shorter.

My thinking as I undid that work was:
1. Is it really okay to do this work at the controller level? Without instantiating a Report model at all?
2. Other reporting methods in the controller rely on the Report model for their filtering, and I think there is some value in consistency.

These aren't the greatest reasons, I recognize. I'll defer to your judgement on whether a one-line controller is more Rails-y - if so, I'm happy to submit that.

(The consistency line of thought could also result in us doing more at the controller level, particularly for list type reports)

Reply:
It definitely confused me into thinking I was missing something when I looked and all the report model did for this was use scopes. I think other reports do more complex data formatting in Report which would make the controllers more complex. In this case, using the Reports actually make the controller more complex and the added abstraction seems more like misdirection than useful. However. Neither is correct and this is not my call, it's yours. When there is not a "right" way, it's up to the person submitting to code to decide on which option to use.